core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -128,14 +128,14 @@ class ReplicaFetcherThread(name: String,
           .format(replica.brokerId, topic, partitionId, followerHighWatermark))
     } catch {
       case e: KafkaStorageException =>
-        fatal("Disk error while replicating data.", e)
+        fatal("Disk error while replicating data for partition %s".format(topicAndPartition), e)
```

Comment:
@ijuma  I have used string interpolation for new changes. I have not changed existing String.format() statements.